### ISSUE TITLE
Pyrogram Asyncio receiving messages throws error

### ISSUE THREAD

**Issue by BitWire** [pyrogram/pyrogram#196]
Hi,
I'm following the official docs with the asyncio version of pyrogram, currently I'm looking at the Update handler, where the official sample gives me this error message: 

`object NoneType can't be used in 'await' expression
Traceback (most recent call last):
  File "/home/tom/.local/lib/python3.6/site-packages/pyrogram/client/dispatcher/dispatcher.py", line 148, in update_worker
    await handler.callback(self.client, *args)
TypeError: object NoneType can't be used in 'await' expression
`
I'm using the sample code from the [https://docs.pyrogram.ml/resources/UpdateHandling](docs) and got the error with both implementations.

**Comment by delivrance** [comment]
@BitWire Hi, what version are you using?
Also show me a snippet of your code and the **full** traceback of the error.

**Comment by BitWire** [comment]
Hi, I'm using v0.10.3.asyncio.
the traceback is complete, my code looks like this:

`from pyrogram import Client, MessageHandler


def my_handler(client, message):
    print(message)


app = Client("my_account")

app.add_handler(MessageHandler(my_handler))

app.run()

`

**Comment by delivrance** [comment]
I agree, even though this issue was not strictly related to Pyrogram itself. You are required some asyncio knowledge if you want to delve into this.

Since all API methods are made asynchronous in the asyncio branch, as a rule of thumb you really just need to put `async` in front of all function definitions that are going to work with coroutines, and use `await` when calling methods. The rest remains basically the same.

The default develop branch is [still using the non-async variant](https://github.com/pyrogram/pyrogram/issues/181); docs will eventually show asyncio examples once I decide to switch.
